tests/workflows/types/test_infer_types.py

```diff
@@ -0,0 +1,110 @@
+from __future__ import annotations
```

Comment:
delete this test file and add only the case we care about here: `src/vellum/workflows/types/tests/test_utils.py::test_infer_types`